github/Repository.py

```diff
@@ -1264,6 +1264,7 @@ def create_git_tag_and_release(
         draft: bool = False,
         prerelease: bool = False,
         generate_release_notes: bool = False,
+        make_latest: str = "true",
```

Comment:
This should be a `bool`:
```suggestion
        make_latest: bool = True,
```

Reply:
alright